cloudflare_worker/worker.js

```diff
@@ -102,15 +110,20 @@ function rolloutGroupHeaders(request){
   return headers
 }
 
-function presortToControl(request, control_cookie_pattern, control_group){
+function presortToControl(request, control_cookie_pattern, control_group, killswitch){
   let responseObj = {
     assignment: '',
     isNew: false,
   }
 
-  // If a user has a control cookie, keep them in the control group
+  // If the killswitch is enabled just return
```

Comment:
This code doesn't seem necessary.  If the switch is on it doesn't matter what this returns because we will never use this object.

Reply:
:+1: